src/overlays/actors/ovl_Shot_Sun/z_shot_sun.c

```diff
@@ -14,7 +33,17 @@ void ShotSun_Init(Actor* thisx, GlobalContext* globalCtx);
 void ShotSun_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void ShotSun_Update(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void ShotSun_FairyCountdown(ShotSun* this, GlobalContext* globalCtx);
+void ShotSun_StartFairyCountdown(ShotSun* this, GlobalContext* globalCtx);
+void func_80BADF0C(ShotSun* this, GlobalContext* globalCtx);
+void ShotSun_HyliaSunUpdate(ShotSun* this, GlobalContext* globalCtx);
+
+extern s32 func_8005B198();
+extern void func_80062718(ColliderCylinder* collider, Vec3s* pos);
+extern void func_80078884(u16 sfxId);
```

Comment:
`extern` isn't required for function prototypes.
```suggestion
s32 func_8005B198();
void func_80062718(ColliderCylinder* collider, Vec3s* pos);
void func_80078884(u16 sfxId);
```